package.json

```diff
@@ -6,7 +6,7 @@
     "lint": "tools/check_tidy.py && eslint .",
     "test": "npm run lint && npm run test-jerry && npm run test-iotjs",
     "test-jerry": "deps/jerry/tools/run-tests.py --unittests",
-    "test-iotjs": "tools/build.py --run-test --no-check-valgrind",
+    "test-iotjs": "tools/build.py --run-test",
```

Comment:
@algebrait CI doesn't use this to enable/disable valgrind, and I think we might try to open it at [this line](https://github.com/Rokid/ShadowNode/blob/master/tools/travis_script.py#L27)?